src/main/java/domain/lotto/LottoTicket.java

```diff
@@ -45,4 +45,27 @@ private void sortByAsc(List<Integer> lottoNumbers) {
     public String toString() {
         return lottoNumbers.toString();
     }
+
+    public boolean contains(Integer winningNumber) {
+        return lottoNumbers.contains(winningNumber);
+    }
+
+    public List<Integer> getLottoNumbers() {
+        return lottoNumbers;
+    }
+
+    public int contains(LottoTicket winningTicket) {
+        int match = 0;
+        for (Integer winningNumber : winningTicket.getLottoNumbers()) {
+            match = getMatch(winningNumber, match);
+        }
+        return match;
+    }
```

Comment:
```suggestion
    public int contains(LottoTicket winningTicket) {
        return lottoNumbers.stream()
                .filter(winningTicket::contains)
                .toList()
                .size();
    }
```
이와 같이 수정하면 getMatch() 메소드와 getLottoNumbers() 메소드를 이용하지 않아도 될 것 같습니다

Reply:
이 부분도 단위테스트를 작성해주시면 좋을 것 같습니다!

Reply:
오오 쩌는데요? 당장 사용해보겠슴다 ㅎㅎ